keras_nlp/tokenizers/word_piece_tokenizer_test.py

```diff
@@ -170,6 +173,87 @@ def test_from_file(self):
         call_output = tokenizer(input_data)
         self.assertAllEqual(call_output, [[1, 2, 3, 4, 5, 6, 7]])
 
+    @unittest.mock.patch("keras_nlp.tokenizers.word_piece_tokenizer.download")
```

Comment:
This exceeds my expectations! It's nice to have testing of the basic functionality separate from `network_tests/`.